test/unit/org/apache/cassandra/io/util/Files.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.io.util;
+
+import java.nio.file.FileSystem;
+
+import com.google.common.jimfs.Jimfs;
+
+public class Files
+{
+    public static FileSystem newInMemoryFileSystem()
```

Comment:
renamed to `newGlobalInMemoryFileSystem` in https://issues.apache.org/jira/browse/CASSANDRA-18485, if we rename here the rebase should avoid the test failing to setup the global fs (this method exists, but doesn't update `File`

Reply:
Renamed